db/csv/my.timesheet-TeamCalendar.csv

```diff
@@ -1,2 +1,2 @@
 YEAR
-2020
+2022
```

Comment:
[pp] Also here it would be nice to put the current year automaticallty. Perhaps a script `createLocalData` setting it would make sense.

Reply:
I added a script to update the date